app/controllers/admin/talentlms_controller.rb

```diff
@@ -0,0 +1,34 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/master/LICENSE.md
+###
+
+module Admin
+  class TalentlmsController < ApplicationController
+    before_action :require_can_manage_config!
+
+    def index
+      @config = Talentlms::Config.first_or_initialize
+    end
+
+    def create
+      @config = Talentlms::Config.first_or_initialize
+      @config.update_attributes(config_params)
```

Comment:
I believe `update_attributes` is equivalent to `update`

Reply:
Yes! Thanks!